### ISSUE TITLE
Light glitch flashes regularly

### ISSUE THREAD

**Issue by cdalexndr** [Aircoookie/WLED#3751]
### What happened?

Every few seconds all LEDs flash quickly, see video:

https://github.com/Aircoookie/WLED/assets/22418250/d4f03a34-96fe-43e9-9338-ba8b599e10b5

### To Reproduce Bug

Solid color, notice flash.

### Expected Behavior

No flashing

### Install Method

Binary from WLED.me

### What version of WLED?

WLED 0.14.1 (build 2401141)

### Which microcontroller/board are you seeing the problem on?

ESP32

### Anything else?

Issue not reproductible with WLED sound reactive 0.13.4.
https://github.com/wled-install/wled-install.github.io/blob/main/bin_dir/2_Sound_reactive_version/0.13.4/WLEDSR_0.13.4_esp32_4MB_M.bin

WS2815 leds:
![WLED_settings](https://github.com/Aircoookie/WLED/assets/22418250/33c8d629-6329-4491-bdbc-da4357987371)
[cfg.json](https://github.com/Aircoookie/WLED/files/14192979/cfg.json)

esp-wroom-32 board

Related: #230

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blazoncek** [comment]
HomeAssistant? MQTT? Alexa? Hue?
Any of those? If yes, then try to disable and retry.

**Comment by cdalexndr** [comment]
After more testing, the glitch also happens with 0.13.4, but more rarely.

Indeed, HomeAsssistant WLED Integration was the culprit.
Disabling "Enable polling for updates" solved this issue.
![image](https://github.com/Aircoookie/WLED/assets/22418250/ec1fc867-6e44-40f7-b456-d83a4b9fa932)

HA wled integration diagnostics: 
[config_entry-wled-6c600e5f3b03226e216b08a97e0179ca.json.txt](https://github.com/Aircoookie/WLED/files/14201800/config_entry-wled-6c600e5f3b03226e216b08a97e0179ca.json.txt)

**Comment by blazoncek** [comment]
Instead of polling ask HA developers to subscribe to websockets for updates.

**Comment by cdalexndr** [comment]
Opened [issue on HA ](https://github.com/home-assistant/core/issues/109914) on HA side

**Comment by Panoramiac** [comment]
I might have the same issue here. In my case the end of a 364 SK6812 RGBW LED strip from BTF-LIGHTING  (about 12m) starts to flicker heavy from time to time. I now disabled the polling in HA. I will observe it and provide and update (including a video). It is powered by a 30A power supply with separate power injections and works since 2 years, so I was thinking more about a SW bug instead of a hardware issue as it was not touched since then.

**Comment by jnpatrick99** [comment]
This bug has been in WLED 0.13+ firmware for quite some time. Personally I gave up and stopped using WLED because if it a year ago, and just bought a finalized HW/SW solution from 3rd party on Amazon.

**Comment by DynamoBen** [comment]
I have the same issue with the releaseed version on on set of Eufy LEDs which use SK6812 RGBW. I noticed this issue during the betas, to correct it I had to load the previous version, factory reset, and recreate all the presets. To troubleshoot I deselected everything in `Sync Interfaces` and deleted the device from HA, still had the flicker. I am using the QuinLED-Dig-Uno with Ethernet if that matters but only running WiFi. Happy to try other things if there are any suggestions.

**Comment by blazoncek** [comment]
The issue comes (as it was written on many occasions) from the resource exhaustion on WLED side. When too many HTTP requests are made ESP cannot handle everything in a manner that would feed LED data consistently (with everything WLED has to offer).
The longer the LED strip the more pronounced or frequent the flashes.

LED data is fed by NeoPixelBus (the underlying driver) and there is little WLED can do.

You can help a little by flashing a firmware with faster CPU and/or flash clock but not all ESPs will handle that gracefully.

**Comment by Panoramiac** [comment]
> The issue comes (as it was written on many occasions) from the resource exhaustion on WLED side. When too many HTTP requests are made ESP cannot handle everything in a manner that would feed LED data consistently (with everything WLED has to offer). The longer the LED strip the more pronounced or frequent the flashes.
> 
> LED data is fed by NeoPixelBus (the underlying driver) and there is little WLED can do.
> 
> You can help a little by flashing a firmware with faster CPU and/or flash clock but not all ESPs will handle that gracefully.

Interesting, are these overclocked images available or would we need to built them by our own? For now I disabled the polling for now and could survive with it. Higher CPU clock might also require a passive cooling element on the chip.

**Comment by blazoncek** [comment]
At most times faster flash speed helps. I never used CPU overclocking though.

Add this to your PIO environment when building:
`board_build.f_flash = 80000000L`

**Comment by Akaricchi** [comment]
See also #3592. I have posted a workaround patch for the HA polling issue there.

**Comment by blazoncek** [comment]
> See also #3592. I have posted a workaround patch for the HA polling issue there.

But you never cared to make a PR so anyone could benefit.

**Comment by Akaricchi** [comment]
> > See also #3592. I have posted a workaround patch for the HA polling issue there.
> 
> But you never cared to make a PR so anyone could benefit.

Hey, at least I cared to publish my findings before I find the time to make a more robust PR that's actually suitable for merging.

**Comment by arildj78** [comment]
In case someone else are looking through this, I will add an observation.

I had this same issue with my setup. Two strings with 300 pixels each and a flicker every 10.0 seconds. I'm currently on 0.15.0 firmware on a MagWLED-1 board with a 5V 90 Watt power supply.

Once in a while I the flicker is just in one of the strings and sometimes it skips a 10 second window, just to appear at the 20 second mark.

Taking the lights off my WiFi stopped the flickering, but it reappeared as soon as I reconnected.
The permanent fix (so far) is to do as @cdalexndr and disable polling. That stopped the flickering as far as I can observe.

I have not tried 0.15.3 but will give it a go after x-mas. I'm not 100% sure about the WLED integration version in Home Assistant, but it seems like its 2025.12.2.

This is the menu where I found the polling setting.
<img width="1155" height="517" alt="Image" src="https://github.com/user-attachments/assets/7edcf077-4165-4a9a-ad8c-57dcaf814cb6" />

This is what was turned off.
<img width="478" height="199" alt="Image" src="https://github.com/user-attachments/assets/e8c5d568-d396-4197-9b72-694cc8d4425d" />
